### PR TITLE
Release 4.4.2

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Review by blackbaud-paulah — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.4.2 (2021-02-11)
+
+- Fixed the toolbar component to have proper visual styles when built for production. [#190](https://github.com/blackbaud/skyux-layout/pull/190)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Fixed the toolbar component's visual styles. [#190](https://github.com/blackbaud/skyux-layout/pull/190)
```
No big deal if you don't want to change this, but the wording felt a little clunky to me.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.4.2](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.4.2)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md:3]: I think I would rather leave this as is. I agree it is a little clunky; however, this issue only affected things in production and didn't really involve changing any styles. I think the clunky version more accurately reflects what the release changed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.4.2](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-release-4.4.2)

File: CHANGELOG.md (hunk 1 above)
Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Sounds good. Thanks!

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**